x-pack/plugins/ml/public/application/data_frame_analytics/common/use_results_view_config.ts

```diff
@@ -54,6 +64,29 @@ export const useResultsViewConfig = (jobId: string) => {
           analyticsConfigs.data_frame_analytics.length > 0
         ) {
           const jobConfigUpdate = analyticsConfigs.data_frame_analytics[0];
+          const analysisType = getAnalysisType(jobConfigUpdate.analysis);
+          // don't fetch the total feature importance if it's outlier_detection
+          if (
+            analysisType === ANALYSIS_CONFIG_TYPE.CLASSIFICATION ||
+            analysisType === ANALYSIS_CONFIG_TYPE.REGRESSION
```

Comment:
In `plugins/ml/common/util/analytics_utils.ts` we have `isClassificationAnalysis` and `isRegressionAnalysis` which we could use here.

Reply:
I have updated it to using the util functions here https://github.com/elastic/kibana/pull/78238/commits/e18bc414d34f7ca2611b2fa0cbcf0a2403d980a4